client/src/components/InputOutput.css

```diff
@@ -6,30 +6,41 @@
 	padding: 0;
 }
 
+.input-output-wrapper {
+	display: flex;
+	width: 100%;
+	/* min-height: 65vh; */
+}
+
 .input-container {
+	background-color: #d7dcce;
+}
+
+.input-wrapper {
 	display: flex;
 	flex-direction: column;
 	width: 100%;
-	height: 60vh;
+	height: 100%;
+	background-color: #d7dcce;
 }
 
-.all-output-container {
-	height: 350px;
+.output-container {
+	height: 100%;
+}
+
+.output-wrapper {
+	background-color: #d7dcce;
+	height: 100%;
+	display: flex;
+	flex-direction: column;
 }
-/* 
-  .volume-copy-container {
-    display: flex;
-    position: absolute;
-    right: 10px;
-    top: 36%;
-  } */
 
 .input-field,
 .output-field {
 	display: flex;
-	padding: 30px;
+	padding: 40px;
 	width: 100%;
-	height: 100vh;
```

Comment:
I think it better you use for Height: 100vh; because when you are using vh in means when user work with the other screens it can cover the whole screen :-)